numpy/core/numeric.py

```diff
@@ -804,7 +804,7 @@ def convolve(a, v, mode='full'):
     -----
     The discrete convolution operation is defined as
 
-    .. math:: (a * v)[n] = \\sum_{m = -\\infty}^{\\infty} a[m] v[n - m]
+    .. math:: (a * v)_n = \\sum_{m = -\\infty}^{\\infty} a_m v_{n - m}
```

Comment:
Maybe make this consistent with the formula above?
```suggestion
    .. math:: c'_n = \\sum_{m = -\\infty}^{\\infty} a_m v_{n - m}
```

Reply:
Actually `np.convolve` doesn't introduce any `c`. Also the notation with `*` is pretty standard in mathematical texts, so I think it's clearer to keep it this way. What do you think?